### ISSUE TITLE
数据集图文相似行打分貌似没有什么用：Taiyi-CLIP-RoBERTa-102M-ViT-L-Chinese 我测试打分不搭嘎的情况返回的还是1

### ISSUE THREAD

**Issue by gg22mm** [IDEA-CCNL/Fengshenbang-LM#403]
先用IDEA-CCNL/Taiyi-CLIP-RoBERTa-102M-ViT-L-Chinese对这两个数据集的图文对相似性进行打分，取CLIP Score大于0.2的图文对作为我们的训练集。
有个疑问：
打分模型Taiyi-CLIP-RoBERTa-102M-ViT-L-Chinese 我测试打分返回的是1，我看图片与文字完全不搭嘎也返回1.0

-------------------------

from PIL import Image
import requests
import open_clip # pip install open_clip_torch
import torch
from transformers import BertModel, BertConfig, BertTokenizer
from transformers import CLIPProcessor, CLIPModel
import numpy as np

query_texts = ['徐师傅太子锅图片 - 第4张']  # 这里是输入文本的，可以随意替换。
text_tokenizer = BertTokenizer.from_pretrained("D:/fenxi/大语言问答模型/ChatRWKV/画图/Taiyi-CLIP-RoBERTa-102M-ViT-L-Chinese")
text_encoder = BertModel.from_pretrained("D:/fenxi/大语言问答模型/ChatRWKV/画图/Taiyi-CLIP-RoBERTa-102M-ViT-L-Chinese").eval()

url = "https://gimg2.baidu.com/image_search/src=http%3A%2F%2Fi1.s2.dpfile.com%2Fpc%2Fc82b7bf4259c1d2910def9fc95ed4785%28700x700%29%2Fthumb.jpg&refer=http%3A%2F%2Fi1.s2.dpfile.com&app=2002&size=f9999,10000&q=a80&n=0&g=0n&fmt=jpeg?sec=1630758762&t=2664f0d13657efa0abb9ff0a86584a48"  # 这里可以换成任意图片的url

clip_model, _, processor = open_clip.create_model_and_transforms('ViT-L-14', pretrained='openai')
clip_model = clip_model.eval()

text = text_tokenizer(query_texts, return_tensors='pt', padding=True)['input_ids']
image = processor(Image.open(requests.get(url, stream=True).raw)).unsqueeze(0)
with torch.no_grad():
    image_features = clip_model.encode_image(image)
    text_features = text_encoder(text)[1]
    # 归一化
    image_features = image_features / image_features.norm(dim=1, keepdim=True)
    text_features = text_features / text_features.norm(dim=1, keepdim=True)
    # 计算余弦相似度 logit_scale是尺度系数
    logit_scale = clip_model.logit_scale.exp()
    logits_per_image =  logit_scale * image_features @ text_features.t()
    logits_per_text = logits_per_image.t()
    probs = logits_per_image.softmax(dim=-1).cpu().numpy()
    print(np.around(probs, 3))

#返回如下：但是看上面的图片地址与说明完全不搭嘎
(mygpt) E:\wll\phpsys\WWW\start\l__dzkj_test\test\resnet\tool\图片相似度>python 图文相似度.py
[[1.]]
